zenoh_app/parse_payload.py

```diff
@@ -0,0 +1,13 @@
+from zenoh_ros_type.autoware_adapi_msgs import ChangeOperationMode, ResponseStatus
+
+
+def ChangeOperationMode_payload(payload: bytes) -> ChangeOperationMode:
+    success_payload = payload[1]
```

Comment:
Since [the PR is merged](https://github.com/evshary/zenoh-ros-type-python/pull/3), now you can use class directly.

Reply:
Maybe you can refer to the usage [here](https://github.com/evshary/zenoh_autoware_fms/blob/ec5b320e179184ed2e077cdac3b0eca3f1991758/zenoh_app/teleop_autoware.py#L37).
The reason why we need to put it to zenoh_ros_type is that we can deserialize it directly to class. Then you don't need to calculate the offset of payload.

Reply:
I've made the changes by adding deserialize function and removing the payload parsing file